cmd/ssl-expiry-check/main.go

```diff
@@ -64,7 +92,9 @@ func main() {
 	reportErr := reportKHSuccess()
 	if reportErr != nil {
 		log.Error(reportErr)
+
 	}
+	os.Exit(1)
```

Comment:
If it reaches this point, it should be considered `successful` -- you should just exit with `0` here

Reply:
Ack, updated for next commit.